crossChain.json

```diff
@@ -2770,6 +2770,64 @@
       "execute": "300000"
     }
   },
+  {
+    "name": "Metis Andromeda Mainnet",
+    "chainId": 1088,
+    "type": "mainnet",
+    "confirmations": 3,
+    "network": "metis",
+    "domainId": "1835365481",
+    "nativeCurrency": {
+      "name": "METIS",
+      "symbol": "METIS",
+      "decimals": 18
+    },
+    "assetId": {
```

Comment:
What about `TEST` deployed on these mainnets ?

Reply:
Good catch! Need to add supported tokens here.